CMakeLists.txt

```diff
@@ -340,14 +340,20 @@ endif()
 # Go down the tree.
 #
 
+if(EXISTS "${DEPS_PREFIX}/share/nvim-qt")
+  option(BUNDLE_NVIMQT "Bundle neovim-qt" ON)
```

Comment:
any reason not to use the existing convention: https://github.com/neovim/neovim/blob/cb757f2663e6950e655c6306d713338dfa66b18d/cmake.deps/cmake/BuildLuarocks.cmake#L99

Reply:
I used that pattern initially.  Then I thought since this variable is in a different cmake project (`USE_BUNDLED_XXX` is in 'cmake.deps' project), maybe the pattern should be different.  But maybe I was thinking too much.

Will change and push soon.